app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/asset/AssetMessageTypes.kt

```diff
@@ -270,12 +271,16 @@ fun getDownloadStatusText(assetDownloadStatus: Message.DownloadStatus, assetUplo
         assetDownloadStatus == Message.DownloadStatus.SAVED_INTERNALLY -> stringResource(R.string.asset_message_downloaded_internally_text)
         assetDownloadStatus == Message.DownloadStatus.DOWNLOAD_IN_PROGRESS ->
             stringResource(R.string.asset_message_download_in_progress_text)
-        assetDownloadStatus == Message.DownloadStatus.SAVED_EXTERNALLY
-                || assetUploadStatus == Message.UploadStatus.UPLOADED -> stringResource(R.string.asset_message_saved_externally_text)
+        assetDownloadStatus == Message.DownloadStatus.SAVED_EXTERNALLY ||
+                assetUploadStatus == Message.UploadStatus.UPLOADED -> stringResource(R.string.asset_message_saved_externally_text)
         assetDownloadStatus == Message.DownloadStatus.FAILED_DOWNLOAD -> stringResource(R.string.asset_message_failed_download_text)
         else -> ""
     }
 
+@Composable
+private fun canNotClick(assetDownloadStatus: Message.DownloadStatus, assetUploadStatus: Message.UploadStatus) =
```

Comment:
minor thing, but I think the usual approach to flag is to have a prefix "is" so perhaps "isClickable" will be more fitting, your call here

Reply:
Agree, renamed !